src/main/java/seedu/duke/commands/DoneCommand.java

```diff
@@ -0,0 +1,24 @@
+package seedu.duke.commands;
+
+import seedu.duke.trackables.Task;
+
+import java.util.List;
+
+public class DoneCommand extends Command {
+
+    private int taskId;
+
+    public DoneCommand(int taskId) {
+        this.taskId = taskId - 1;
+    }
+
+    @Override
+    public void execute(List<Task> tasks) {
+        tasks.get(taskId).markAsDone();
```

Comment:
Maybe you want to assign the task that you have retrieved to a variable so that later you can directly call toString() on it? This will then prevent from doing two get(), which means traversing twice through the list, which can increase the complexity of the solution.